lib/Views/RPlan.dart

```diff
@@ -69,13 +69,20 @@ class RPlan extends State {
   }
 
   // Get Data
-  Future loadRPlan() async{
+  Future loadRPlan(int dayManager) async{
+    api_models.VPlan vplan = await API.of(context).requests.getVPlan(searchedTeacher, dayManager);
+    if (vplan.date.weekday == 6) {          // if Saturday, show the day after tomorrow
+      dayManager = 2;
+    } else if (vplan.date.weekday == 7) {   // if Sunday show tomorrow
+      dayManager = 1;
+    }
     if (_loaded == 3) _loaded = 0;
     _days = [];
-    _loadDay(0);
-    _loadDay(1);
+    // _loadDay: 0 = today, 1 = tomorrow, 2 = the day after tomorrow
+    _loadDay(dayManager);
+    _loadDay(dayManager + 1);
     if (hasTeacherPlan) {
-      _loadDay(2);
+      _loadDay(dayManager + 2);
```

Comment:
Funktioniert das denn überhaupt?
Ich denke besser wäre es `getVPlanTime` in `lib/api/api_helpers.dart` zu bearbeiten, oder hier einfach 0 nicht am Wochenende zu laden.